test/fixtures/generic/encounter.json

```diff
@@ -8,6 +8,11 @@
         "Annual_Physical": {
             "type": "Encounter",
             "wellness": true,
+            "codes": [{
```

Comment:
Was this added because something broke otherwise? We don't require codes for wellness encounters